src/cards/jennDiazCard/jennDiazCard.jsx

```diff
@@ -1,33 +1,28 @@
 import React from 'react';
 import './jennDiazCard.scss';
-import imageFront from './jenn.png';
-import imageBack from './jenn-on-bike.png';
+import headshot from './jenn.png';
```

Comment:
Can you cut the image file size? We're only rendering 120px so should be able to minimize it a bit

Reply:
@jischr any recommendations on how to do it quickly? what tool do you use

Reply:
I just used Preview in Mac to resize! It's really quick.

Reply:
oh! I have previously used `sips` in the command line to resize images, make it super easy especially if you want to resize a bunch of images as once. 

https://ss64.com/osx/sips.html
https://medium.com/shell-life/convert-images-for-web-using-the-sips-command-line-on-macosx-656c502a67a6

I can push a commit to resize the image, Is there a size we want to maintain for images moving forward?